sympy/functions/special/error_functions.py

```diff
@@ -2078,7 +2077,15 @@ def _eval_aseries(self, n, args0, x, logx):
                     for k in range(int(n/2) - 1)] + [Order(1/z**n, x)]
             return (sin(z)/z)*Add(*p) - (cos(z)/z)*Add(*q)
 
-        # All other points are not handled
+        # Expansion at -oo
+        elif point is S.NegativeInfinity:
+            z = self.args[0]
+            p = [S.NegativeOne**k * factorial(2*k) / z**(2*k)
+                    for k in range(int((n - 1)/2))] + [I*pi] + [Order(1/z**n, x)]
+            q = [S.NegativeOne**k * factorial(2*k + 1) / z**(2*k + 1)
+                    for k in range(int(n/2) - 1)] +[Order(1/z**n, x)]
+            return (sin(z)/z)*Add(*p) - (cos(z)/z)*Add(*q)
+
```

Comment:
I think there is too much repetition of code here. 

Use this rather 
```
if point in (S.Infinity, S.NegativeInfinity):
     // write the code which i think is same in both cases
    // store in a variable like result
    if point is -oo:
        // Add the extract I*pi to result
    // return result
```

Hope that makes sense

Reply:
okay sir